### ISSUE TITLE
[React] Revamp Landing Page for #50DaysOfJavaScript

### ISSUE THREAD

**Issue by vinitshahdeo** [vinitshahdeo/50DaysOfJavaScript#5]
## Current State
The [current landing page](https://50daysofjavascript.netlify.app/) for **50DaysOfJavaScript** needs a significant overhaul. It should be transformed into a single-page application (SPA) with clear Calls to Action (CTAs) to encourage visitors to join as participants, mentors, and community members.

## Proposed Changes

- Revamp the existing landing page into a single-page application to provide a seamless and interactive user experience.

- **CTAs for User Engagement**: Incorporate prominent and visually appealing CTAs that guide visitors to join as participants, mentors, or become a part of the community.

- **Initiative Overview**: Include a concise description of the initiative, emphasizing that it is a community-led open-source effort aimed at helping individuals enhance their JavaScript coding skills. Ensure that it conveys the message that the daily coding challenges are practical and relevant to the everyday tasks of JavaScript developers.

- **Link to Join**: Provide a clear and accessible link for users to join `#50DaysOfJavaScript`. This link can be placed prominently within the initiative's overview section.

- **Social Media Sharing**: Add an option in the footer to allow users to easily share the initiative on Twitter. This can help increase the visibility of the initiative among potential participants.

## Additional Information

The landing page should be designed with user-friendliness and engagement in mind. It should effectively communicate the value and purpose of the initiative, making it enticing for visitors to get involved.

## Related Links

- [Join #50DaysOfJavaScript](https://tinyurl.com/50DaysOfJavaScript)
- [Landing Page](https://50daysofjavascript.netlify.app/)

## Note

**Please share the UI mocks in this issue thread before starting the development.**

**Comment by deeksharungta** [comment]
Hey, can I work on this issue?

**Comment by amitkroutthedev** [comment]
@vinitshahdeo Can I help with this issue?

**Comment by deeksharungta** [comment]
![image](https://github.com/vinitshahdeo/50DaysOfJavaScript/assets/79518532/c92af267-9424-4641-84b4-6a5d2c225e9f)
Hey, I want to work on this issue and as required,  I've created a UI design for the landing page.

**Comment by vinitshahdeo** [comment]
Hello @deeksharungta 

The design looks awesome 🎉 !

Just one minor suggestion, Can we have a bit of coding-js-related-illustrations in the background? This will be inline with the #50DaysOfJavaScript theme.

**Comment by vinitshahdeo** [comment]
Thanks for showing your interest @amitkroutthedev! Could you please share your design or collaborate with @deeksharungta to take it forward?

**Comment by amitkroutthedev** [comment]
Ok

**Comment by deeksharungta** [comment]
> Hello @deeksharungta
> 
> The design looks awesome 🎉 !
> 
> Just one minor suggestion, Can we have a bit of coding-js-related-illustrations in the background? This will be inline with the #50DaysOfJavaScript theme.

Yes, sure, I'll try to add some coding-related illustrations in the background.

**Comment by deeksharungta** [comment]
@vinitshahdeo I've added some coding-related illustrations and created two variants. Please let me know if either of these fits your requirements, and I'll be happy to make any necessary changes.




![Screenshot from 2023-09-11 13-07-33](https://github.com/vinitshahdeo/50DaysOfJavaScript/assets/79518532/2abe333b-3faa-486e-be55-855890030990)


![Screenshot from 2023-09-11 13-08-01](https://github.com/vinitshahdeo/50DaysOfJavaScript/assets/79518532/d5e2b37b-8da5-43f2-9677-fe9dfde15ce7)

**Comment by vinitshahdeo** [comment]
This looks awesome @deeksharungta 🎉 
Thanks for sharing the designs.

Let's catch up for a quick call to discuss it further—please [block my calendar](https://peerlist.io/vinitshahdeo) to reduce the friction of async communication.

**Comment by amitkroutthedev** [comment]
Should I join?

**Comment by deeksharungta** [comment]
> This looks awesome @deeksharungta 🎉 Thanks for sharing the designs.
> 
> Let's catch up for a quick call to discuss it further—please [block my calendar](https://peerlist.io/vinitshahdeo) to reduce the friction of async communication.

I've scheduled a meeting for 8:00 am on the 17th. It would be much appreciated if we could reschedule it for the evening, depending on your availability, as there weren't any other time options available.

**Comment by vinitshahdeo** [comment]
@amitkroutthedev Yes, please.

**Comment by vinitshahdeo** [comment]
Thanks, @deeksharungta!
I shared an updated invite. I hope this works. Please lemme know if you wanna reschedule this.

💼 [50DaysOfJavaScript]
📆 Sunday, September 17 · 8:00 – 9:00pm
⌛ Time zone: Asia/Kolkata

@amitkroutthedev Please share email - I will add you to the invite.

**Comment by amitkroutthedev** [comment]
@vinitshahdeo amitroutadev.0723@gmail.com

**Comment by deeksharungta** [comment]
> Thanks, @deeksharungta! I shared an updated invite. I hope this works. Please lemme know if you wanna reschedule this.
> 
> 💼 [50DaysOfJavaScript] 📆 Sunday, September 17 · 8:00 – 9:00pm ⌛ Time zone: Asia/Kolkata
> 
> @amitkroutthedev Please share email - I will add you to the invite.


Thanks for the updated invite; the schedule works for me.
